test/integration/lib/storages/RedisProxyCache.int.test.ts

```diff
@@ -73,14 +80,14 @@ describe('RedisProxyCache Integration Tests -->', () => {
   };
 
   describe('Use Cases Tests with redis cluster -->', () => {
-    const proxyCache = createProxyCache(STORAGE_TYPES.redisCluster, redisClusterProxyConfig);
-    const anotherProxyCache = createProxyCache(STORAGE_TYPES.redisCluster, redisClusterProxyConfig);
+    const proxyCache = createProxyCache(STORAGE_TYPES.redisCluster, redisClusterProxyConfig) as IProxyCache<RedisClientType>;
```

Comment:
I think, `createProxyCache` factory should create needed type of _proxyCache_ only based on _proxyConfig_, we shouldn't explicitly specify the type using `as`